### ISSUE TITLE
CloudFormation Script fails on s3 bucket creation

### ISSUE THREAD

**Issue by donhenton** [aws-samples/amazon-ecs-mythicalmysfits-workshop#10]
Maybe I missed something but I wasn't able to press the magic button on the setup section. I got a 'permissions denied' error on the bucket creation. I think this may be due to the change in the way s3 buckets can be configured for website (public access).

To get that script to work I did this:

* pull json from the designer
* modify the output section,adding condition to S3WebsiteURL section:
```
 "S3WebsiteURL": {
            "Description": "This is the DNS name of your S3 site",
            "Value": {
                "Fn::GetAtt": [
                    "MythicalBucket",
                    "WebsiteURL"
                ]
            },
	    "Condition": "MakeBucket"
        }
```
* run the script with MakeBucket false
* create an s3 website capable bucket
* in the cloudnine environment run setup script passing in unique s3 name of website bucket

### when performing cleanup, lots of things failed, this may help, before deleting stack:

* remove all images from the repositories
* stop all services on the clusters
* wait a long time, (1 hr) still will get failures mostly with the IPs and VPC but they can be deleted manually

Hope this helps somebody. This was a *VERY* useful tutorial

**Comment by jasonumiker** [comment]
My theory is that you had this turned on which blocked the bucket creation with public permissions. Turning it off before deploying the stack should do it. I had that off and it works for me in Oregon. https://aws.amazon.com/blogs/aws/amazon-s3-block-public-access-another-layer-of-protection-for-your-accounts-and-buckets/

**Comment by hub714** [comment]
Yeah a couple things changed from when we initially created this workshop. We'll look into it.

**Comment by jarednielsen** [comment]
Same issue here. Would love help on a workaround.

**Comment by hub714** [comment]
@jarednielsen The bucket should create successfully now with CloudFormation. Where you might still run into issues is when you upload an object. Where did you run into the problem?

**Comment by jarednielsen** [comment]
The bucket creates successfully, but I'm unable to upload an object because of blocked public access. Is there a workaround for an account with blocked public access?
